contacts24/contacts.py

```diff
@@ -31,23 +32,33 @@ def parse_input(user_input: str) -> tuple[str, Optional[CommandArguments]]:
     return cmd, *args
 
 
+# def load_contacts_book() -> AddressBook:
```

Comment:
@krisdzhus  if you don't need code - delete it, don't comment for PR
We have git, where we can check any version and rollback any change from past

No need to comment blocks